packages/eslint-plugin/src/rules/no-magic-numbers.ts

```diff
@@ -3,8 +3,8 @@ import {
   AST_NODE_TYPES,
 } from '@typescript-eslint/experimental-utils';
 import baseRule from 'eslint/lib/rules/no-magic-numbers';
+import { JSONSchema4 } from 'json-schema'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Admittedly more ceremony, but thinking out loud - would it be cleaner to refactor to export this type from the experimental-utils package given it actually depends on it?

Reply:
I'll remove this usage. it shouldn't be needed. This was just the rule author working around the fact that he knew something was an array when the types say it could not be.